pkg/render/logstorage/dashboards/dashboards.go

```diff
@@ -49,6 +50,18 @@ var (
 	PolicyName               = networkpolicy.TigeraComponentPolicyPrefix + Name
 )
 
+// GetJobName makes a unique job name per operator version.
+// For dev images it takes the first 32 chars.
+func GetJobName() string {
+	operatorVersion := strings.ReplaceAll(version.VERSION, ".", "-")
+
+	if len(operatorVersion) >= 32 {
```

Comment:
What is "32" in this context? Why do we need to truncate? I think we should have a comment explaining why we do this - it's not obvious!